src/serializer/Referentializer.js

```diff
@@ -41,23 +41,23 @@ export class Referentializer {
     realm: Realm,
     options: SerializerOptions,
     scopeNameGenerator: NameGenerator,
-    referentializedNameGenerator: NameGenerator
+    scopeBindingNameGenerator: NameGenerator
   ) {
     this._options = options;
     this.scopeNameGenerator = scopeNameGenerator;
+    this.scopeBindingNameGenerator = scopeBindingNameGenerator;
 
     this.referentializationState = new Map();
-    this._referentializedNameGenerator = referentializedNameGenerator;
```

Comment:
This doesn't seem to be referenced anywhere or do anything. Just wanted to flag this out and make sure it really can be removed safely